VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -227,7 +227,7 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
     }
 
     async retrieveAudioStreamInfo(hearingId): Promise<void> {
-        if (this.conference.status === ConferenceStatus.InSession || this.participant.status === ParticipantStatus.InConsultation) {
+        if (this.conference.status === ConferenceStatus.InSession) {
```

Comment:
Was there no unit test to update for this change?

Reply:
no, I was expecting some test to fail. I am really not sure how this was implemented, the test suggest that this was added by mistake, but the commit and the ticket that was associated with this code change seems like it was expected to show that alert in consultation room

Reply:
It looks like we never assert on conferenceRecordingInSessionForSeconds changing. But it's run so the code coverage stays high, could we add a couple of tests for this while it's being worked on? Or it may be possible to include it in the expecations of an existing one.

Reply:
added couple of tests to cover this now